### PR TITLE
Added receivedServiceMessage in Connection.ts. The message can be any messages from service that are not being processed by SpeechRecognizer, Intent, Translator or Dialog.

### PR DESCRIPTION
**Pull request by lisaweixu**

### CODE REVIEW

File: src/sdk/ConnectionEventArgs.ts [hunk 1]
```diff
@@ -11,3 +11,11 @@ import { SessionEventArgs } from "./Exports";
...
 export class ConnectionEventArgs extends SessionEventArgs {
 }
+
+/**
+ * Defines payload for connection events like Connected/Disconnected.
+ * Added in version 1.9.0
+ */
+// tslint:disable-next-line:max-classes-per-file
+export class ServiceEventArgs extends SessionEventArgs {
+}
```
Comment (rhurey) [inline src/sdk/ConnectionEventArgs.ts]: Why is this here and again in ServiceEventArgs.ts?

File: src/sdk/Connection.ts [hunk 2]
```diff
@@ -74,12 +83,14 @@ export class Connection {
      * Closes the connection the service.
      * Users can optionally call closeConnection() to manually shutdown the connection of the associated Recognizer.
      *
-     * If closeConnection() is called during recognition, recognition will fail and cancel wtih an error.
+     * If closeConnection() is called during recognition, recognition will fail and cancel with an error.
```
Comment (rhurey) [inline src/sdk/Connection.ts]: Thanks

File: src/sdk/Connection.ts [hunk 3]
```diff
@@ -74,12 +83,14 @@ export class Connection {
...
      */
     public closeConnection(): void {
         this.privServiceRecognizer.disconnect();
     }
 
+    public receivedServiceMessage: (args: ServiceEventArgs) => void;
```
Comment (rhurey) [inline src/sdk/Connection.ts]: Comment.

File: src/common/ConnectionEvents.ts [hunk 4]
```diff
@@ -5,6 +5,21 @@ import { ConnectionMessage } from "./ConnectionMessage";
 import { IStringDictionary } from "./IDictionary";
 import { EventType, PlatformEvent } from "./PlatformEvent";
 
+// tslint:disable-next-line:max-classes-per-file
```
Comment (rhurey) [inline src/common/ConnectionEvents.ts]: The first class in the file shouldn't need this.

**Comment by rhurey** [comment]
>                                     errorCallBack);

Does this need to raise the events also?

---
Refers to: src/common.speech/DialogServiceAdapter.ts:673 in 65996b0. [](commit_id = 65996b0b80ad747d45d8249063c07debf3ca1fd9, deletion_comment = False)

File: tests/Settings.ts [hunk 5]
```diff
@@ -8,6 +8,9 @@ export class Settings {
     public static SpeechRegion: string = "<<YOUR_REGION>>";
     public static SpeechEndpoint: string;
 
+    public static VoiceSignatureEnrollmentEndpoint: string = "wss://10.91.185.105:8446/speech/voicesignature?&language=en-us";
```
Comment (rhurey) [inline tests/Settings.ts]: >10.91.185.105:8446 [](start = 68, length = 18)

Can you open a task so we replace this with a public URL once the endpoint is in prod?

File: src/sdk/ConnectionEventArgs.ts (hunk 1 above)
Comment (lisaweixu) [reply src/sdk/ConnectionEventArgs.ts]: removed

File: src/common/ConnectionEvents.ts (hunk 4 above)
Comment (lisaweixu) [reply src/common/ConnectionEvents.ts]: removed

File: tests/Settings.ts (hunk 5 above)
Comment (lisaweixu) [reply tests/Settings.ts]: Bug 2214615: JavaScript: Add VoiceSignatureEnrollment PPE endpoint into tests/Settings.ts
